webrender/src/device.rs

```diff
@@ -1169,7 +1202,34 @@ impl Device {
             }
         }
 
-        gl::bind_framebuffer(gl::FRAMEBUFFER, self.default_fbo);
+        // TODO(gw): Hack! Modify the code above to use the normal binding interfaces the device exposes.
+        gl::bind_framebuffer(gl::READ_FRAMEBUFFER, self.bound_read_fbo.0);
```

Comment:
While I do see this code being correct, I can't shake off the feeling that something is wrong, given that the old code was binding a different FBO. So the new code will behave differently. If it still works, perhaps the current FBO gets overwritten by something later?